pkg/log/logrus.go

```diff
@@ -0,0 +1,45 @@
+package log
+
+import "github.com/sirupsen/logrus"
```

Comment:
Instead of the log package importing logrus, this should just be done in dex's `main`, and passed as a logger.

Reply:
See my comment bellow.